src/Packetery/Module/Carrier/OptionsPage.php

```diff
@@ -236,7 +236,10 @@ private function createForm( array $carrierData ): Form {
 		$form->addHidden( 'id' )->setRequired();
 		$form->addSubmit( 'save' );
 
-		if ( false === $carrier->hasPickupPoints() && in_array( $carrier->getCountry(), Carrier::ADDRESS_VALIDATION_COUNTRIES, true ) ) {
+		if ( false === $carrier->hasPickupPoints()
```

Comment:
Uprav prosím formátování podmínky, abychom to měli jednotné - každá podmínka na svém řádku a && na konci řádků.

Reply:
Rozumim, opraveno.